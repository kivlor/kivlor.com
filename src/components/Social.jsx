import * as React from 'react';

export const Social = () => (
  <nav className="links">
    <ul>
      <li><a className="gitlab" href="http://gitlab.com/kivlor" target="_blank">gitlab</a></li>
      <li><a className="github" href="http://github.com/kivlor" target="_blank">github</a></li>
      <li><a className="instagram" href="http://instagram.com/kivlor" target="_blank">instagram</a></li>
    </ul>
  </nav>
);
