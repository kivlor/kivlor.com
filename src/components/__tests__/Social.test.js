import React from 'react';
import renderer from 'react-test-renderer';

import { Social } from '../Social';

describe('Social', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Social />);

    expect(component).toBeTruthy();
  });

  it('matches snapshot', () => {
    const component = renderer.create(<Social />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
