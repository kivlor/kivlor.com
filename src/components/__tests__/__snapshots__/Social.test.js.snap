// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social matches snapshot 1`] = `
<nav
  className="links"
>
  <ul>
    <li>
      <a
        className="github"
        href="http://github.com/kivlor"
        target="_blank"
      >
        github
      </a>
    </li>
    <li>
      <a
        className="instagram"
        href="http://instagram.com/kivlor"
        target="_blank"
      >
        instagram
      </a>
    </li>
    <li>
      <a
        className="twitter"
        href="http://twitter.com/kivlor"
        target="_blank"
      >
        twitter
      </a>
    </li>
  </ul>
</nav>
`;
